demos/src/Examples/Images/React/index.jsx

```diff
@@ -19,8 +19,8 @@ export default () => {
     ],
     content: `
       <p>This is a basic example of implementing images. Drag to re-order.</p>
-      <img src="https://source.unsplash.com/8xznAGy4HcY/800x400" />
-      <img src="https://source.unsplash.com/K9QHL52rE2k/800x400" />
+      <img src="https://placehold.co/600x400" />
+      <img src="https://placehold.co/800x400" />
```

Comment:
tests were failing because these images are no longer accessible

Reply:
thx, fixed on main

Reply:
rebased it out 👍